### PR TITLE
LPS-102097 - Deactivating compatibility layer for grid

### PR DESCRIPTION
**Pull request by marcoscv-work**
This PR turns off the BS3 compatibility layer for Grid
I've changed for that:

- img old grid clases
- col old clases
- containers old classes

I've split commerce changes in its own commits

---

To test the PR:

- Navigate along portal (especially commerce) and see containers still working fine

---

It is coming from:
https://github.com/liferay-frontend/liferay-portal/pull/497

### CODE REVIEW

File: modules/apps/frontend-theme/frontend-theme-admin/src/css/_clay_variables.scss [hunk 1]
```diff
@@ -23,6 +23,7 @@ $compat-buttons: false;
 $compat-cards: false;
 $compat-figures: false;
 $compat-form_validation: false;
+$compat-grid: false;
```
Comment (jbalsas) [inline modules/apps/frontend-theme/frontend-theme-admin/src/css/_clay_variables.scss:26]: Let's do more than this, let's **remove** it from `styled` so we don't need to do it here. See my [comments](https://issues.liferay.com/browse/LPS-102092?focusedCommentId=2314006&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-2314006) about the real goal of these tasks.

Comment (marcoscv-work) [reply modules/apps/frontend-theme/frontend-theme-admin/src/css/_clay_variables.scss:26]: Ok !!!!

Comment (marcoscv-work) [reply modules/apps/frontend-theme/frontend-theme-admin/src/css/_clay_variables.scss]: https://github.com/liferay-frontend/liferay-portal/pull/499/commits/4a3843ef20c557f81bc6737b1d35469475df81d6

**Comment by marcoscv-work** [comment]
ci:test:bundle

**Comment by marcoscv-work** [comment]
ci:test:stable

**Comment by marcoscv-work** [comment]
ci:test:sf

File: util-taglib/src/com/liferay/taglib/ui/UserDisplayTag.java [hunk 2]
```diff
@@ -59,7 +59,7 @@ public int doStartTag() throws JspException {
 				String.valueOf(_displayStyle));
 
 			if (Validator.isNull(_imageCssClass)) {
-				_imageCssClass = "img-circle";
+				_imageCssClass = "circle";
```
Comment (eduardoallegrini) [inline util-taglib/src/com/liferay/taglib/ui/UserDisplayTag.java:62]: `.rounded-circle` ?

**Review by eduardoallegrini — APPROVED** [review]
As we spoke this morning, I agree with you on merging this PR and then fix the markup

We should consider these tasks globally:
- Remove nested containers `.container > .container`
- Add the missing row and cols `.container > .row > .col`
- Update Bootstrap space utilities to Clay `.mx-auto -> .c-mx-auto`
- Update layout class to taglibs `.container -> <clay:layout:container>`

File: util-taglib/src/com/liferay/taglib/ui/UserDisplayTag.java (hunk 2 above)
Comment (marcoscv-work) [reply util-taglib/src/com/liferay/taglib/ui/UserDisplayTag.java:62]: right! update an rebase!

**Comment by marcoscv-work** [comment]
Rebased and fix the rounded class!

**Comment by marcoscv-work** [comment]
ci:test:sf

**Comment by marcoscv-work** [comment]
ci:test:bundle

**Comment by marcoscv-work** [comment]
success ci:test:stable

**Comment by marcoscv-work** [comment]
ci:test:stable

**Comment by jbalsas** [comment]
Hey @marcoscv-work, just FYI, and unless I'm mistaken, you need to run `ci:test:relevant` to forward PRs, not `ci:test:stable`.

**Comment by marcoscv-work** [comment]
@john-co the PR should be ready to be sent to Brian, any other consideration? test or something?
As Chema told us, that perfect time to make more aggressive changes

**Comment by marcoscv-work** [comment]
ci:test:sf

**Comment by marcoscv-work** [comment]
ci:test:relevant

**Comment by marcoscv-work** [comment]
ci:test:sf

**Comment by jbalsas** [comment]
>We should consider these tasks globally:
>
>Remove nested containers .container > .container
>Add the missing row and cols .container > .row > .col
>Update Bootstrap space utilities to Clay .mx-auto -> .c-mx-auto
>Update layout class to taglibs .container -> <clay:layout:container>

Hey @eduardoallegrini, would you mind creating those tasks and adding them to the Epic? 🙏

**Comment by jbalsas** [comment]
ci:test:sf

**Comment by jbalsas** [comment]
ci:test:relevant

**Comment by john-co** [comment]
> @john-co the PR should be ready to be sent to Brian, any other consideration? test or something?
> As Chema told us, that perfect time to make more aggressive changes

Agreed. Nothing else at this point, as long as `ci:test:relevant` passes and the test plan from description is done.

**Comment by john-co** [comment]
asked about errors in https://liferay.slack.com/archives/C19PWQ74J/p1605566564170100

**Comment by john-co** [comment]
ci:test:relevant

**Comment by marcoscv-work** [comment]
rebased

**Comment by marcoscv-work** [comment]
ci:test:relevant

**Comment by pat270** [comment]
LGTM

**Comment by marcoscv-work** [comment]
ci:test:relevant

**Comment by marcoscv-work** [comment]
After testing issues in https://github.com/liferay-frontend/liferay-portal/pull/505 (initially, it would be the first to be merged) We have decided to merge this one first.

**Comment by marcoscv-work** [comment]
Rebase errors not found.

**Comment by marcoscv-work** [comment]
ci:test:forward

**Comment by eduardoallegrini** [comment]
ci:forward

**Pull request closed without merging**